waspc/data/Generator/templates/server/src/routes/operations/_action.js

```diff
@@ -1,17 +1,22 @@
 {{={= =}=}}
+import { 
+  deserialize as superjsonDeserialize,
+  serialize as superjsonSerialize,
+} from 'superjson'
 import { handleRejection } from '../../utils.js'
-
 {=& operationImportStmt =}
 
 export default handleRejection(async (req, res) => {
-  const args = req.body || {}
+  const args = (req.body && superjsonDeserialize(req.body)) || {}
```

Comment:
Our deserialization process on the server is analogous to the one on the client (https://github.com/wasp-lang/wasp/pull/1090/files#r1159749811).

More precisely:
1. When deserializing, Express will turn `'{"json": "...", "meta": "..." }'` (string) to `{ json: "<serialized_data>", meta: ".<metadata>" }` (object) , and we'll do the rest with `superjsonDeserialize`. The end result is `data`.
2. When serializing, we will turn `data` into `{ json: "<serialized_data>", meta: ".<metadata>" }` (object) using `superjsonSerialize`, and Express will do the rest. The end result is `'{"json": "<serialized_data>", "meta": "<metadata>" }'` (string).